packages/components/src/components/post-icon/post-icon.tsx

```diff
@@ -1,37 +1,95 @@
 import { Component, Host, h, Prop, State } from '@stencil/core';
 
+/**
+ * @class PostIcon - representing a stencil component
+ * 
+ * @param {string} name - The name/id of the icon (e.g. 1000, 1001, ...).
+ * @param {string} base - The base path, where the icons are located (must be a public url).
+ * @param {boolean} flipH - When set to `true`, the icon will be flipped horizontally.
+ * @param {boolean} flipV - When set to `true`, the icon will be flipped vertically.
+ * @param {number} scale - The `number` for the css `scale` transformation.
+ * @param {number} rotate - The `number` of degree for the css `rotate` transformation.
+ * @param {boolean} animation - The name of the animation (`cylon`, `cylon-vertical`, `spin`, `spin-reverse`, `fade`, `throb`).
+ * 
+ * @exports {class} PostIcon - Which will be processed to a custom web-component on build.
+ * 
+ * @see: https://jsdoc.app/ and https://en.wikipedia.org/wiki/JSDoc
+ */
```

Comment:
Take a look at https://github.com/swisspost/design-system/blob/main/packages/internet-header/src/components/post-internet-header/post-internet-header.tsx and https://github.com/swisspost/design-system/tree/main/packages/internet-header/src/components/post-internet-header.

If you put the comment for the props above the props themselves, the "Description" column in the props table will contain the docblock comment.